Module/Rule.Registry/Convert/Functions.SingleLine.ps1

```diff
@@ -390,7 +390,7 @@ function Get-RegistryValueNameFromSLStig
 
     if ($valueName)
     {
-        $valueName = $valueName.Matches.Value -replace '["���]', ''
+        $valueName = $valueName.Matches.Value -replace '[\u201C\u201D]|["���]', ''
```

Comment:
This is little but you don't have to specify the empty string on the replace.

Reply:
When we make a parser change where all the xccdfs will be regenerated, I'll address this at that time.  I like the idea of simplifying it.